dapps/templates/boilerplate/config/contracts.js

```diff
@@ -1,6 +1,8 @@
 module.exports = {
   // default applies to all environments
   default: {
+    library: 'embarkjs',  // can also be 'web3'
```

Comment:
Should we or do we need to do similar in `dapps/tests/` or is it okay for them to just rely on defaults being merged into their configs?

Reply:
it's ok, many ppl wouldn't have this defined, so its' expected